lms/djangoapps/ytp/management/commands/update_user_fullname.py

```diff
@@ -0,0 +1,73 @@
+"""
+Script for update User's first name, last name  and the name field in UserProfile
+"""
+import json
+from logging import getLogger
+
+import requests
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.db.models import Q
+from social_django.models import UserSocialAuth
+
+log = getLogger(__name__)
```

Comment:
log is not a good approach for the manage command, `stdout.write` suits better.

Reply:
I deleted log and used stdout.write.